.github/workflows/helmsman_mosip_destroy.yml

```
@@ -0,0 +1,201 @@
+  destroy:
+    runs-on: ubuntu-latest
+    needs: validate-confirmation
+    environment: 
+      name: ${{ github.ref_name }}
+    env:
+      KUBECONFIG: ${{ github.workspace }}/.kube/config
+      PATH: ${{ github.workspace }}/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+      KUBECTL_PATH: ${{ github.workspace }}/.local/bin/kubectl
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
```

actionlint (1.7.8)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)